libs/estdlib/src/ssl.erl

```diff
@@ -67,15 +67,16 @@
 -export_type([
     sslsocket/0,
     host/0,
-    hostname/0
+    hostname/0,
+    tls_client_option/0
 ]).
 
 -type host() :: hostname() | ip_address().
 -type hostname() :: string().
 -type ip_address() :: inet:ip_address().
 -type tls_client_option() :: client_option().
 -type client_option() ::
-    {server_name_indication, sni()}.
+    {server_name_indication, sni()} | {verify, verify_none}.
```

Comment:
Are `client_options()` also supposed to accept `gen_tcp:options()` here? (See question above)

Reply:
yes, fixed it. Make it compliant with otp.